custom-routes-vis/src/main/webapp/script.js

```diff
@@ -335,15 +335,56 @@ function selectRouteDisplayDetails(
   let routeInfoElement = document.getElementById('route-info');
   routeInfoElement.innerText = 'Selected Route Info:' +
       '\nDuration: ' + formatDuration(totalDurationSec) +
-      '\nDistance: ' + formatDistance(totalDistanceMeters);
+      '\nDistance: ' + formatDistance(totalDistanceMeters) + '\n';
 
   if (routeToken !== '') {
-    routeInfoElement.innerText += '\nRoute Token: check the developer console';
+    let copyBtn = document.createElement('button');
```

Comment:
Full words please.
(replace everywhere)
```suggestion
    let copyButton = document.createElement('button');
```